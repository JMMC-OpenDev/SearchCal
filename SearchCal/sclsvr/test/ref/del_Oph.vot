version https://git-lfs.github.com/spec/v1
oid sha256:49b9dd9e7e9f7a169ec5080bdbef57d5e0ea06751a03feee2e3e7fe48ea3522b
size 129719
