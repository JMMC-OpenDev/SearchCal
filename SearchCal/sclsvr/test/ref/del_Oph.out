version https://git-lfs.github.com/spec/v1
oid sha256:b63c5dbd11ca78ca09de9e9487ae1de3102ab0bcf54ae06675a42e8199ab15bc
size 174039
